### ISSUE TITLE
Not able to change borderRadius on AutoCompleteList

### ISSUE THREAD

**Issue by selvinkuik** [anubra266/choc-autocomplete#67]
I have the following implementation:

```
<AutoComplete openOnFocus>
  <AutoCompleteInput variant="filled" />
  <AutoCompleteList borderRadius="none">
```

However, border radius is not registering. I can confirm, other style props (e.g. fontSize) work.

**Comment by anubra266** [comment]
Could you provide a codesandbox reproduction please?

**Comment by selvinkuik** [comment]
Sure, here you go: https://codesandbox.io/s/choc-ui-packages-codesandbox-forked-76vih?file=/src/App.js

On line 24 notice how `mt` works but both `borderRadius` and `boxShadow` do not

**Comment by selvinkuik** [comment]
Possibly related, but in the codesandbox above, I realise not all style props are pulling through for `AutoCompleteGroupTitle` either

**Comment by anubra266** [comment]
@selvinkuik  Thanks to @Buupu this is fixed in [v4.7.1](https://github.com/anubra266/choc-autocomplete/releases/tag/v4.7.1)
